content/en/docs/reference/glossary/kube-controller-manager.md

```diff
@@ -4,14 +4,14 @@ id: kube-controller-manager
 date: 2018-04-12
 full_link: /docs/reference/command-line-tools-reference/kube-controller-manager/
 short_description: >
-  Component on the master that runs controllers.
+  Component of the Control Plane that runs controllers.
```

Comment:
```suggestion
  Control Plane component that runs controller processes.
```

"Control plane component that runs controllers" sounds a little weird in english, hence borrowing from the description on [line 18](https://github.com/kubernetes/website/pull/18438/files#diff-f876097bd77a5621a4930dab8a3d5540R18) for some rewording.